backend/LexBoxApi/GraphQL/ProjectMutations.cs

```diff
@@ -43,6 +46,8 @@ public record CreateProjectResponse(Guid? Id, CreateProjectResult Result);
             input = input with { ProjectManagerId = loggedInContext.User.Id };
         }
 
+        if (loggedInContext.User.Email == null) throw new ProjectCreatorsMustHaveEmail("Project creators must have an email address");
```

Comment:
This mutation already has `[VerifiedEmailRequired]`, which covers this and more.
What's still missing is having a "user has a verified email" check in both `AddProjectMember` and `ChangeProjectMemberRole`.
```suggestion
```

Reply:
While the VerifiedEmailRequired attibute currently covers this check, I could see us changing its logic at some point in the future. We might at some point think "Wait, why are we setting EmailVerificationRequired = true when there's no email address, that makes no sense" and forget that we were relying on it here to guard against email being null. So I prefer to have the null check here as well.

Reply:
then we should just modify the VerifiedEmailRequired to require the user to have an email, and then we make a comment about it.

Reply:
looking at the code in context, let's add a new permission check for `HasRequestProjectPermission` and we can wrap up whatever we want there, in this case they probably don't have permission if they were created by someone else.

Reply:
Done in commit 79470ff4f7217b06e862dcdf5f8b3de21f09b84a.